website/docs/r/dhcp_server_ip_pool.html.markdown

```diff
@@ -14,7 +14,7 @@ Provides a resource to configure IP Pool for logical DHCP server on NSX-T manage
 
 ```hcl
 data "nsxt_edge_cluster" "edgecluster" {
-  display_name = "edgecluster1"
+  display_name = "EDGECLUSTER1"
```

Comment:
can we use the env variable here and below?

Reply:
Let's just not make any change to the doc file. It's not relevant to this PR

Reply:
Changed back.